### ISSUE TITLE
stropts.h file not found

### ISSUE THREAD

**Issue by jgowdy** [dotnet/coreclr#43]
Are the library dependencies of the CoreCLR Linux build documented somewhere?  I'm getting this when I try to build on CentOS 6.6

[  1%] Building CXX object src/pal/src/CMakeFiles/CoreClrPal.dir/init/pal.cpp.o
/home/dev/coreclr/src/pal/src/init/pal.cpp:63:10: fatal error: 'stropts.h' file not found
#include <stropts.h>
         ^
1 error generated.
make[2]: *** [src/pal/src/CMakeFiles/CoreClrPal.dir/init/pal.cpp.o] Error 1
make[1]: *** [src/pal/src/CMakeFiles/CoreClrPal.dir/all] Error 2
make: *** [all] Error 2
Failed to build coreclr components.

**Comment by jgowdy** [comment]
Fixed the issue with:

touch /usr/include/stropts.h

Is that something we want to fix in the build process though?

**Comment by ellismg** [comment]
Presently we only build on and test a very specific configuration for Linux, which is Ubuntu 14.04.  We plan to include support for building on other distributions and versions at a later date, but right now we're focused on one specific version as we bring the stack up.

I think we'd rather wait on addressing these issues until we get to a point where we have validation in our CI system so we know we won't regress.

**Comment by jgowdy** [comment]
That makes sense.  Is there somewhere that we've documented what the current specific configuration is?  I saw in another issue something about only supporting Clang 3.5.  I would suggest a section of the readme.md specifying what the supported platform for Linux build is presently, because plenty of people are going to try to build it on Linux (hopefully) and if they're trying on some distro other than Ubuntu or they're using one of the many distros still shipping Clang 3.4, it's going to seem like this simply doesn't work.

I'd make a pull request for changes to readme.md but I don't know what the requirements are beyond Ubuntu 14.04 and Clang 3.5.  Is that it?

**Comment by jgowdy** [comment]
"Once the engineering support (e.g. build and validation) is functionally complete, we will share the details on how to build and test the product on Linux."

I see like many people I'm jumping the gun here based on the above language, but I'd recommend throwing something out there because most of the curious people like myself are going to likely try to build it anyway and discover the requirements from the issue responses or lose interest.  I'm pretty motivated personally to see this catch on.

**Comment by jgowdy** [comment]
I was able to successfully build with a fresh install of minimal Ubuntu Server 14.04, stock cmake 2.8.12.2, and Clang 3.5.  Just putting that much information should be sufficient for most people.

**Comment by pgavlin** [comment]
Fixed by #82.
